src/html/modules/home/map.html

```diff
@@ -4,72 +4,6 @@ <h2 class="map__main-title">See examples of our work across three continents</h2
       We can use technology and big data to enable on-the-ground partners to save the world’s most threatened
       rainforests and habitats. Saving rainforests isn’t only the key to halting climate change, It’s also vital
       to supporting many of the world’s poorest communities who rely on rainforests for food, shelter and livelihood.
-      <span class="map__main-descr-notice">(click on icon to switch country)</span>
   </p>
-  <div class="map__content">
-    <div class="map__area">
-      <img class="map__img" src="images/map.jpg" alt="">
-      <span class="icon-logo map__point map__point_peru js-map-point active" style="left: 23%; top: 74%;" data-name="peru"></span>
-      <span class="icon-logo map__point map__point_ecuador js-map-point" style="left: 20%; top: 68%;" data-name="ecuador"></span>
```

Comment:
Please delete the CSS (from the .less file) of the HTML that you deleted.

Reply:
Good point.